diagnostic.md

```diff
@@ -14,7 +17,8 @@ list items (`li`) using jQuery? This is intentionally vague. (Assume starting
 at the `OL` DOM node).
 
 <!-- your answer starts here -->
-
+$('li').each('body');
```

Comment:
`.each()` is an iterator method that runs a callback function for each element in the jQuery array-like object.  This will select all `<li>` elements and attempting to run the string `body` on each of them.